### PR TITLE
85 add llm service provider

### PR DESCRIPTION
**Pull request by dominhnhut01**
Add llm services, network services, token usage service, and  get env variable service to new app

### CODE REVIEW

**Review by TranHVLoc — COMMENTED** [review]
Lot of modules contain plain code without any comment.

File: new-chatbot-app/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "format:check": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\"",
```
[the remark this answers is not in the document]
Comment (dominhnhut01) [reply new-chatbot-app/package.json:11]: Based on Agile Development, comment should be left at the end of the development process

[the remark this answers is not in the document]
Comment (TranHVLoc) [reply new-chatbot-app/package.json:11]: Got it! But each function should be separated by at least 1 line of comment to clarify what it does

**Review by TranHVLoc — APPROVED** [review]

File: new-chatbot-app/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dominhnhut01) [reply new-chatbot-app/package.json:11]: > Should we do `format:write` and `formate:check`?

Done

**Pull request merged**